src/dmclock/src/dmclock_server.h

```diff
@@ -50,6 +50,7 @@
 #include "profile.h"
 #endif
 
+#include "include/assert.h"
```

Comment:
i think that changes under src/dmclock need to go through the dmclock repo instead. cc @ivancich

Reply:
Since the dmclock library exists outside of ceph and can be used outside of ceph, we would not want to make this change within the library itself.

So we have to decide whether to remove all these changes to dmclock_server.h, or the alternative would be to maintain this diff into the future.

Is it a problem if a library ceph uses calls `assert(false)`?